docs/Streams.md

```diff
@@ -186,3 +194,21 @@ MsQuic does not enforce it, and it is legal for an application to provide less b
 
 After the initial receive window is full, flow control will ensure that the peer does not send more data than there is buffer space available.
 However, the application should still provide enough buffer space to keep flow control from impacting performances.
+
+## Receive Shutdown
+
+The receiver can abortively shutdown a stream receive direction by calling [`StreamShutdown`](api/StreamShutdown.md) 
+with the `QUIC_STREAM_SHUTDOWN_FLAG_ABORT_RECEIVE` option.
+
+# Closing a Stream
+
+Once a stream has been shutdown (on both direction for a bi-directional stream), the application receives a
```

Comment:
I think it reads better as "in both directions"

Reply:
typo aside, this reads much better than the previous wording!

Reply:
```suggestion
Once a stream has been shutdown (in both direction for a bi-directional stream), the application receives a
```